src/tools/recipe2plan-cli.ts

```diff
@@ -68,7 +68,8 @@ async function main() {
     Runtime.init('../..');
     fs.mkdirSync(opts.outdir, {recursive: true});
 
-    const plans = await recipe2plan(opts._[0], outFormat, opts.recipe);
+    const manifest = await Runtime.parseFile(opts._[0]);
```

Comment:
Do you need to wrap this in a `Flags.withDefaultReferenceMode`?

Reply:
hmmm... I don't know. You may have more context here than I do. What's the thinking here?

Reply:
AFAIU, everything within the scope of this function will use reference mode stores. I assume that we'd want to include manifest parsing with this state? It's a bit tricky to catch, since our tests are set up to use this, but it's different from how we run the tool.

Reply:
ok, added

Reply:
Thanks!